bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookKeeperAdminTest.java

```diff
@@ -61,7 +61,7 @@
     public BookKeeperAdminTest() {
         super(numOfBookies, 480);
         baseConf.setLostBookieRecoveryDelay(lostBookieRecoveryDelayInitValue);
-        baseConf.setOpenLedgerRereplicationGracePeriod(String.valueOf(30000));
+        baseConf.setOpenLedgerRereplicationGracePeriod(500);
```

Comment:
is this change required? does it affect anyhow? for the sake of simplicity cannt we just leave it to default value

Reply:
well the point of having this change is to reduce the test time on this test case. 30 seconds is too much for a test... why do we need want to test 30 seconds...

Reply:
ok..i forgot that in this test testDecommissionForLedgersWithMultipleSegmentsAndNotWriteClosed, there are not writeclosed ledgers and they will remain underreplicated for openLedgerRereplicationGracePeriod.